translations/wesnoth-A_New_Order/es.po

```diff
@@ -21923,185 +22510,167 @@ msgstr ""
-msgstr ""
+msgstr "Traducción al húngaro:"
 
 #. [entry]
 #: A_New_Order/about.cfg:141
 msgid "Italian translation:"
-msgstr ""
+msgstr "Traducción al italiano:"
 
 #. [entry]
 #: A_New_Order/about.cfg:144
 msgid "Polish translation:"
-msgstr ""
+msgstr "Traducción al polaco:"
 
 #. [entry]
 #: A_New_Order/about.cfg:147
 msgid "Russian translation:"
-msgstr ""
+msgstr "Traducción al ruso:"
 
 #. [entry]
 #: A_New_Order/about.cfg:150
 msgid "Spanish translation:"
-msgstr ""
+msgstr "Traducción al español:"
 
 #. [entry]
 #: A_New_Order/about.cfg:153
 msgid "Swedish translation:"
-msgstr ""
+msgstr "Traducción al sueco:"
 
 #. [entry]
 #: A_New_Order/about.cfg:156
 msgid "Turkish translation:"
-msgstr ""
+msgstr "Traducción al turco:"
 
 #. [about]
 #: A_New_Order/about.cfg:160
 msgid "Additional thanks to"
-msgstr ""
+msgstr "Agradecimientos adicionales"
 
 #. [entry]
 #: A_New_Order/about.cfg:210
 msgid "all others from the Wesnoth forums"
-msgstr ""
+msgstr "todos los demás del foro de Wesnoth"
 
 #. [about]
 #: A_New_Order/about.cfg:215
 msgid "Campaign funding"
-msgstr ""
+msgstr "Financiación de la campaña"
 
 #. [about]
 #. I had to shorten this to get it to fit:
 #: A_New_Order/about.cfg:238
 msgid "More info may be found within readme.txt file"
-msgstr ""
+msgstr "Más informacion en el fichero readme.txt"
 
 #. [entry]
 #: A_New_Order/about.cfg:240
 msgid "(as described previously)"
-msgstr ""
+msgstr "(como fue descrito)"
 
 #. [about]
 #: A_New_Order/about.cfg:245
 msgid "I can't believe you actually read this thing"
-msgstr ""
+msgstr "No puedo creer que realmente estés leyendo esto"
 
 #. [entry]
 #. I wanted to include "in BfW 1.16, now that Wesnoth has made changes to how it handles credits" in this string,
 #. but that got cut off for me... if your language can fit more info in less space, feel free to include that part:
 #: A_New_Order/about.cfg:249
 msgid "(although it's a bit more understandable now)"
-msgstr ""
+msgstr "(aunque se entiende un poco mejor ahora)"
 
 #. [about]
 #: A_New_Order/about.cfg:253
 msgid "Darth Vader"
-msgstr ""
+msgstr "Darth Vader"
 
 #. [entry]
 #. please translate Skywalker literally (as "sky-walker")
 #: A_New_Order/about.cfg:256
 msgid "Father of Luke Skywalker"
-msgstr ""
+msgstr "Padre de Luke Skywalker"
 
 #. [about]
 #. Żwirek i Muchomorek are from a Czech movie for children. You are free to replace them with characters from a favourite foreign movie for children in your country:
 #: A_New_Order/about.cfg:261
 msgid "Żwirek i Muchomorek"
-msgstr ""
+msgstr "Żwirek i Muchomorek"
 
 #. [entry]
 #: A_New_Order/about.cfg:263
 msgid "Famous Czech gay actors"
-msgstr ""
+msgstr "Actores chechos gay famosos"
 
 #. [about]
 #. Polish president (from 2010 to 2015), he can't spell. Famous for dozens of idiotic sentences and sayings. You are free to replace him with some politician from your country:
 #: A_New_Order/about.cfg:268
 msgid "Bronisław Komorowski"
-msgstr ""
+msgstr "Bronisław Komorowski"
```

Comment:
I kinda think this should be Nicolás Maduro in Spanish

Reply:
Oh, WOW, for that we have a perfect name in Spain: Mariano Rajoy xD

Reply:
ok I googled "Mariano Rajoy gaffe" and it looks like there are plenty of results, so, that should work!